source/includes/steps-upgrade-shared-cluster.yaml

```
@@ -0,0 +1,54 @@
+title: Shutdown All Running Emulators
+ref: add-the-sign-in-with-apple-capability
+content: |
+   Before upgrading your Cluster end any client application's Sync Connection.
```

Reading "end" here blended into the first clause ("upgrade your cluster end, ..."). Comma suggested (or use another word like "close").
```suggestion
   Before upgrading your cluster, end any client application's sync connection.
```